assets/js/src/config.js

```diff
@@ -0,0 +1,8 @@
+/* global wcgaiData */
+export const config = wcgaiData.config ?? {};
+export const events = wcgaiData.events ?? {};
+export const cart = wcgaiData.cart ?? {};
+export const products = wcgaiData.products ?? {};
+export const product = wcgaiData.product ?? {};
+export const addedToCart = wcgaiData.added_to_cart ?? {};
+export const order = wcgaiData.order ?? {};
```

Comment:
:nail_care: 
Could be rewritten to a tad shorter version.
It also minimizes a bit better 5 864b vs 6023b
```suggestion
/* global wcgaiData */
export const {
	config = {},
	events = {},
	cart = {},
	products = {},
	product = {},
	addedToCart: added_to_cart = {},
	order = {},
} = wcgaiData;
```

Reply:
:grey_question: :nail_care: 
 I also wonder, wouldn't it be safer and cleaner to leave `undefined` where it is?
If PHP didn't expose something, let further JS code be aware of it. Right now this file adds a masking layer. 
Further JS code can do `?.` or `Object.values( events ?? {} ).forEach(` as we do.

But if we avoid setting `= {}` here, we could have an insight in JS that PHP didn;t deliver something, and react accordingly.

Reply:
Good point, I've added the [suggested changes](https://github.com/woocommerce/woocommerce-google-analytics-integration/blob/d6259842a405e9387b537a38a60c54ec48ddb168/assets/js/src/config.js).